### PR TITLE
Add conda env to avit prompt

### PR DESCRIPTION
**Pull request by mcleonard**
This adds the conda environment in the avit theme's prompt, inline with the current directory and git info.

Code based on this PR for the agnoster prompt: https://github.com/robbyrussell/oh-my-zsh/pull/7067.

### CODE REVIEW

**Comment by mcleonard** [comment]
Looks like so:

Screen Shot 2019-08-09 at 1 13 34 PM

**Comment by John-P** [comment]
You may want to check out the performance and style suggestions made by @avrahamshukron here: https://github.com/ohmyzsh/ohmyzsh/pull/7067#discussion_r388542761

File: themes/avit.zsh-theme [hunk 1]
```diff
@@ -12,6 +12,20 @@ local _current_dir="%{$fg_bold[blue]%}%3~%{$reset_color%} "
...
 
+# Anaconda: current working conda env
+# Code based on this commit: https://github.com/robbyrussell/oh-my-zsh/pull/7067/files
+# To replace the conda modification of PS1 do `conda config --set changeps1 False`
+# or add 'changeps1: False' to your .condarc file.
+function _conda_env() {
+  local conda_env="$CONDA_DEFAULT_ENV"
+  if [[ -n $conda_env ]]; then
+    local conda_changeps1="`conda config --show changeps1`" 
+    if [[ $conda_changeps1 == "changeps1: False" && $conda_env != "base" ]]; then
```
Comment (avrahamshukron) [inline themes/avit.zsh-theme]:
```suggestion
    if [[ -z $CONDA_PROMPT_MODIFIER ]]; then
```

File: themes/avit.zsh-theme [hunk 2]
```diff
@@ -12,6 +12,20 @@ local _current_dir="%{$fg_bold[blue]%}%3~%{$reset_color%} "
...
+      echo "%{$fg[yellow]%}($CONDA_DEFAULT_ENV)%{$reset_color%} "
```
Comment (avrahamshukron) [inline themes/avit.zsh-theme]: You already have a local var `$conda_env`. Use it.
```suggestion
      echo "%{$fg[yellow]%}($conda_env)%{$reset_color%} "
```

**Pull request closed without merging**